text-show-instances.cabal

```diff
@@ -257,7 +262,8 @@ test-suite spec
                        -- Only defines tests if not using Windows
                        Spec.System.Console.TerminfoSpec
                        Spec.System.PosixSpec
-  build-depends:       base-compat-batteries >= 0.10   && < 1
+  build-depends:       aeson                 >= 1.5.6  && < 2
```

Comment:
Did you mean to impose an upper version bound of `aeson < 2` here? This conflicts with the `aeson >= 2` version bounds in the `library` stanza.